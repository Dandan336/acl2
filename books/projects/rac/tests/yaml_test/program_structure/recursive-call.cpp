// RAC begin

int foo();

int foo()
{
  return foo();
}

// RAC end
