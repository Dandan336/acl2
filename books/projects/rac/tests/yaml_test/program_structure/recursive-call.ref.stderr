recursive-call.cpp:3: syntax error, unexpected ';', expecting '{'
